### ISSUE TITLE
KER (v1.1.0.1) part highlight label showing when hovering over part info from partlist

### ISSUE THREAD

**Issue by UBERKalti** [CYBUTEK/KerbalEngineer#84]
Hello, maybe not so much as a bug but it still can be considered inconvenient.

When using KER v1.1.0.1 on an otherwise stock install of KSP pre-release (v1.1.0.1183) in the VAB and/or SPH, KER's part-infolabel/highlight is shown on top of the partinfo-label when hovering over a part in the partlist, making it hard to read the info. See attached screenshot for issue depiction.

![kerpartlabel](https://cloud.githubusercontent.com/assets/3638347/14247439/82e4de14-fa6e-11e5-9d56-781bb2c76d14.jpg)

**Comment by Jakednd** [comment]
confirming this issue is still present in the 1.2-pre recompile

**Comment by Emilgardis** [comment]
Bug present for me aswell. I believe it is also the cause for certain windows greying out, eg engineers report in VAB.

**Comment by Gerry1135** [comment]
> I believe it is also the cause for certain windows greying out, eg engineers report in VAB.

No, that also happens in installs that don't have KER installed.
